mode

```diff
@@ -0,0 +1,31 @@
+import paddle
```

Comment:
This is not how you how implement a function. Can you kindly share the todo list